### DISCUSSION TITLE
Are there any cases for production?

### DISCUSSION THREAD

**Discussion by zqabc** [Ananto30/zero#19] [General]
A great project.
I am currently investigating microservices. Can this project be used in a production environment? 
ths!

**Comment by Ananto30** [comment]
Hi @rwlrwlrwlrwl thanks!

I don't think it's been used to production yet. I have tried to mimic but cannot ensure stability in case of outage or crashes.

**Reply by zqabc** [reply]
Thank you for your reply.
Look forward to your complete test.

**Comment by martincpt** [comment]
A few years has spent. How about now? 

Furthermore, how would you implement a authentication on top of zero?

**Comment by Ananto30** [comment]
@martincpt I don't know if anyone is using this for production or not, I don't have the data.

Authentication is not implemented yet. Contributions are welcome.
